### DISCUSSION TITLE
spOccupancy feature parity, etc...

### DISCUSSION THREAD

**Discussion by tpoisot** [PoisotLab/SpeciesDistributionToolkit.jl#37] [General]
From @gottacatchenall in #30 

> On a separate note, is the goal here to eventually have this ensemble of packages become something like SpeciesDistributionModels.jl? It seems a suite of simple OTS models with methods for computing validation metrics and visualization (plus some of the occupancy models from MetapopulationDynamics.jl) would put us at feature parity with spOccupancy

**Comment by tpoisot** [comment]
The point of the package is *not* to become `SpeciesDistributionModels.jl`, though this might be something we built at some point. My endgame at the moment is more towards integration with `MLJ.jl`, because (i) this has a mechanism to register new models defined in any package and (ii) a lot of the models we want to use are ML models anyway.

The other obvious advantage is that MLJ has the validation and tuning built-in.

In terms of visualization, I am slowly moving towards compatibility with `Makie.jl`, for a bunch of reasons including the existence of `GeoMakie.jl`.

**Reply by gottacatchenall** [reply]
makes sense, the integration might even be the place to include a batch generator that enforces class balance

**Reply by tpoisot** [reply]
Yeah - some of the best spatial folding work as far as I know is done by @BlasBenito in [`spatialRF`](https://www.blasbenito.com/project/spatialrf/), and it would make sense to rely on this heavily as the basis for spatially-constrained class balance.
